libs/payments/paypal/src/lib/paypal.manager.ts

```diff
@@ -83,6 +83,7 @@ export class PayPalManager {
    */
   async getCustomerPayPalSubscriptions(customerId: string) {
     const subscriptions = await this.stripeManager.getSubscriptions(customerId);
+    if (!subscriptions.data) return [];
```

Comment:
Does `getSubscriptions` ever return an api response with data as falsy? I thought it'd always be an empty array at least.